a8-launcher.py

```diff
@@ -3060,22 +3060,22 @@ def main():
             a8_Logger.traceEnabled = (not Reflect.field(config,"quiet"))
             # src/a8/launcher/Main.hx:132
             if Reflect.field(config,"showHelp"):
-                haxe_Log.trace(a8_launcher_Main.helpString(),_hx_AnonObject({'fileName': "Main.hx", 'lineNumber': 133, 'className': "a8.launcher.Main", 'methodName': "main"}))
```

Comment:
Change --l-help print from trace to print so that the 'Main:XX' class:lineNumber prefex will not be present